cmd/harmony/main.go

```diff
@@ -229,21 +230,56 @@ func setupInitialAccount() (isLeader bool) {
 	return isLeader
 }
 
-func setupConsensusKey(nodeConfig *nodeconfig.ConfigType) *bls.PublicKey {
-	consensusPriKey, err := blsgen.LoadBlsKeyWithPassPhrase(*blsKeyFile, blsPassphrase)
+func readMultiBlsKeys(nodeConfig *nodeconfig.ConfigType) error {
+	multiKeyFolder := "./hmy/multikeys"
```

Comment:
coordinate with @Daniel-VDM as we unify the bls locations here and in go-sdk's hmy cli